sdk/servicebus/Microsoft.Azure.ServiceBus/src/Message.cs

```diff
@@ -467,8 +467,12 @@ internal short PartitionId
             /// <summary>Gets or sets the original sequence number of the message.</summary>
```

Comment:
```suggestion
            /// <summary>Gets the original sequence number of the message.</summary>
```

If this is intended to be read-only, as the remarks detail, we may want to consider clarifying in the summary as well, since that's the typical area that gets read by developers using code-completion.